### PR TITLE
VIH-6234 dismiss witness from hearing (non-ui)

### PR DESCRIPTION
**Pull request by shaed-parkar**
### JIRA link (if applicable) ###

VIH-6234

### Change description ###

* dismiss witness from hearing (non-ui)

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by RocheL — APPROVED** [review]

File: VideoWeb/VideoWeb.UnitTests/Controllers/ConferenceManagement/DismissWitnessTests.cs [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        {
+            var participant = TestConference.Participants.First(x => x.Role == Role.Individual);
+            var user = new ClaimsPrincipalBuilder()
+                .WithUsername(participant.Username)
+                .WithRole(AppRoles.CitizenRole).Build();
+
+            Controller = SetupControllerWithClaims(user);
+
+            var result = await Controller.DismissWitnessAsync(TestConference.Id, participant.Id);
+            var typedResult = (UnauthorizedObjectResult) result;
```
Comment (levibutler) [inline VideoWeb/VideoWeb.UnitTests/Controllers/ConferenceManagement/DismissWitnessTests.cs:35]: could assert type before casting .Should().BeOfType<T>()

Comment (levibutler) [reply VideoWeb/VideoWeb.UnitTests/Controllers/ConferenceManagement/DismissWitnessTests.cs:35]: if changing, same in `should_return_accepted_when_participant_is_witness_and_judge_is_in_conference`

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/services/video-call.service.ts [hunk 2]
```diff
@@ -252,4 +252,12 @@ export class VideoCallService {
         });
         await this.apiClient.callWitness(conferenceId, participantId).toPromise();
     }
+
+    async dismissParticipantFromHearing(conferenceId: string, participantId: string) {
+        this.logger.info(`${this.loggerPrefix} Attempting to dismiss participant from hearing`, {
+            conference: conferenceId,
+            participant: participantId
+        });
+        await this.apiClient.dismissWitness(conferenceId, participantId).toPromise();
```
Comment (levibutler) [inline VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/services/video-call.service.ts:261]: why toPromise? not returning?

**Review by levibutler — APPROVED** [review]

File: VideoWeb/VideoWeb.UnitTests/Controllers/ConferenceManagement/DismissWitnessTests.cs (hunk 1 above)
Comment (shaed-parkar) [reply VideoWeb/VideoWeb.UnitTests/Controllers/ConferenceManagement/DismissWitnessTests.cs:35]: done

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/services/video-call.service.ts (hunk 2 above)
Comment (shaed-parkar) [reply VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/services/video-call.service.ts:261]: The global error handler will capture errors and log it but actually maybe we should capture it to when the request fails and we want to show a message of some nature

**Pull request merged**